app/controllers/ExclusionListController.scala

```diff
@@ -454,7 +454,9 @@ class ExclusionListController @Inject()(
       RegistrationList(None, List(RegistrationItem(iabdType, active = false, enabled = false)))
 
     Logger.info(
-      s"Committing Exclusion for scheme ${request.empRef.toString} , with employees Optimisitic Lock: ${excludedIndividual.get.perOptLock}")
+      s"[ExclusionListController][searchResultsHandleValidResult] Committing Exclusion for scheme ${request.empRef.toString}" +
+        s", with employees Optimisitic Lock: ${excludedIndividual.get.perOptLock}"
```

Comment:
.get on an option. Any way to get around this?

Reply:
didn't notice the .get, will check and update if required